Hw391 week4

duke-aipi-Han opened the pull request:
notebook for Han Wang's exploratory data analysis. This includes a shell script that will pull Welltory dataset from their GitHub into this project. The Jupyter notebook has a line that will run this script that is commented out in case this needs to be setup in a new environment.

duke-aipi-Han committed:
initial checkin of script to get data and exploratory data analysis notebook

calicartels commented:
Hey Han,

We have a  strict policy on the number of files to commit (for this assignment, it is supposed to 1) and I notice extra changes in the commit. Unfortunately that would be a minimum credit. 

Apart from that you also have PR'd it into the main branch, which again is a minimum credit violation.

Work on this next time

duke-aipi-Han committed:
remove extra shell script and update notebook to pull data file directly.

duke-aipi-Han commented:
I updated the PR to be off week4, and removed the extra shell script (and combined it into the notebook). To run the notebook, please uncomment the line that will load the data files.
